lib/kitchen/provisioner/chef_base.rb

```diff
@@ -131,8 +131,8 @@ def install_options
           :omnibus_url => config[:chef_omnibus_url],
           :project => project.nil? ? nil : project[1],
           :install_flags => config[:chef_omnibus_install_options],
-          :use_sudo => config[:sudo],
-          :sudo_command => config[:sudo_command]
+          :use_sudo => false,
```

Comment:
`:use_sudo` actually defaults to `true` in Mixlib::Install which is why I was explicitly setting it to false, avoiding a double-sudo. This isn't really a problem unless someone is setting a wacky `sudo_command`.  I'll make the change, test it on a few platforms, and push it up.  I'll hold-off on squashing the commits in case there is any other feedback from the other maintainers.